dash/_callback.py

```diff
@@ -60,14 +61,14 @@ def is_no_update(obj):
 # pylint: disable=too-many-locals
 def callback(
     *_args,
-    background=False,
-    interval=1000,
-    progress=None,
-    progress_default=None,
-    running=None,
-    cancel=None,
-    manager=None,
-    cache_args_to_ignore=None,
+    background: bool = False,
+    interval: int = 1000,
+    progress: Optional[Output] = None,
+    progress_default: Any = None,
+    running: Optional[List[Tuple[Output, Any, Any]]] = None,
```

Comment:
took me a moment to parse this :-)

Reply:
I think you can just stop at a higher level `Optional[List]`, and be les concrete.